app.freelens.Freelens.yaml

```diff
@@ -0,0 +1,111 @@
+id: app.freelens.Freelens
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: freelens
+separate-locales: false
+finish-args:
+  # X11 performance
+  - --share=ipc
+  # We need X11
+  - --socket=fallback-x11
+  # Access to wayland
+  - --socket=wayland
+  # Audio Access
+  - --socket=pulseaudio
+  # Access to all files
+  - --filesystem=host
+  # All devices (camera, microphone for calls)
+  - --device=all
+  # User's files
+  - --filesystem=~/.freelens:rw
+  - --filesystem=~/.kube:rw
```

Comment:
these should be `--persist` instead.

Reply:
Kubectl does not use XDG Base Directory. $HOME/.kube/config is a default location.
The application also uses $HOME/.freelens as a place to save cache and own extensions.

Reply:
then you should use persist instead.

Reply:
Ok, for ~/.freelens, I can't for ~/.kube because it'll break the ecosystem: Freelens uses the context from host and applications for authorization from host, like ie. `aws eks get-token`. Also it brings the terminal when you could use other tools for Kubernetes installed in the account: ie. kctx, kns, krew, etc.

Reply:
Ah, no, I think I'm wrong and it works all fine with `--persist=~/.kube`

Reply:
Yes, I can confirm: application is broken with `persist=~/.kube` as I just verified.